### PR TITLE
[doctrine/doctrine-bundle] Use dbname_suffix in tests

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| License       | MIT
| Doc issue/PR  | -

Related to https://github.com/doctrine/DoctrineBundle/pull/1328

Diff with 2.3 for 2.4:

```diff
diff -r 2.3/config/packages/test/doctrine.yaml 2.4/config/packages/test/doctrine.yaml
4c4
<         dbname: 'main_test%env(default::TEST_TOKEN)%'
---
>         dbname_suffix: '_test%env(default::TEST_TOKEN)%'
diff -r 2.3/post-install.txt 2.4/post-install.txt
6,8d5
< 
<   * Configure the <fg=green>driver</> (postgresql) and
<     <fg=green>server_version</> (13) in <fg=green>config/packages/doctrine.yaml</>

Only in 2.3: src/Entity and src/Repository
```
```

### CODE REVIEW

**Comment by stof** [comment]
@nicolas-grekas this PR should be marked as draft until the bundle PR is merged, to avoid an auto-merge by the bot

**Review by chalasr — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
This PR is ready /cc @symfony/mergers

**Review by derrabus — APPROVED** [review]

File: doctrine/doctrine-bundle/2.3/manifest.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     },
     "env": {
         "#1": "Format described at https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url",
-        "#2": "IMPORTANT: You MUST configure your server version, either here or in config/packages/doctrine.yaml",
+        "#2": "IMPORTANT: DATABASE_URL *must* define the server version",
```
[the remark this answers is not in the document]
Comment (nicolas-grekas) [reply doctrine/doctrine-bundle/2.3/manifest.json:11]: see https://github.com/symfony/recipes/pull/947

**Comment by GPHemsley-RELX** [comment]
Some of the files introduced in this PR are identical to the existing 2.3 files. Is there a reason they weren't added as symlinks, as is typically done?

**Comment by weaverryan** [comment]
I can’t remember the reasons, but symlinks were causing some issue, so we’re moving away from them

**Pull request merged**